disperser/cmd/encoder/main.go

```diff
@@ -49,16 +51,52 @@ func RunEncoderServer(ctx *cli.Context) error {
 		return err
 	}
 
-	enc, err := NewEncoderGRPCServer(config, logger)
-	if err != nil {
-		return err
+	metrics := encoder.NewMetrics(config.MetricsConfig.HTTPPort, logger)
+	if config.MetricsConfig.EnableMetrics {
+		httpSocket := fmt.Sprintf(":%s", config.MetricsConfig.HTTPPort)
+		metrics.Start(context.Background())
+		logger.Info("Enabled metrics for Encoder", "socket", httpSocket)
 	}
-	defer enc.Close()
 
-	err = enc.Start(context.Background())
+	if config.EncoderVersion == V2 {
+		// We no longer compute the commitments in the encoder, so we don't need to load the G2 points
+		prover, err := prover.NewProver(&config.EncoderConfig, false)
+		if err != nil {
+			return fmt.Errorf("failed to create encoder: %w", err)
+		}
+
+		s3Client, err := s3.NewClient(context.Background(), config.AwsClientConfig, logger)
+		if err != nil {
+			return err
+		}
+
+		blobStoreBucketName := config.BlobStoreConfig.BucketName
+		blobStore := blobstorev2.NewBlobStore(blobStoreBucketName, s3Client, logger)
+		logger.Info("Blob store", "bucket", blobStoreBucketName)
+
+		chunkStoreBucketName := config.ChunkStoreConfig.BucketName
+		chunkWriter := chunkstore.NewChunkWriter(logger, s3Client, chunkStoreBucketName, 4*1024*1024) // TODO(dmanc): make fragment size configurable?
```

Comment:
Not a bad thing to make configurable. That being said, this is unlikely to be something we change often (if ever)... so possibly acceptable to hard code it in the short term.

Reply:
Sounds good, I'll leave it hardcoded and update the comment.

Reply:
For some reason the linter didn't like changing it for one specific file `types_test.go` https://github.com/Layr-Labs/eigenda/actions/runs/11711503781/job/32620653412